### PR TITLE
Put module filter behind toggle

### PR DESCRIPTION
**Pull request by benrudolph**
@amsagoff @snopoke @kaapstorm

### CODE REVIEW

File: corehq/apps/app_manager/tests/test_suite.py [hunk 1]
```diff
@@ -18,6 +20,16 @@
 class SuiteTest(SimpleTestCase, TestFileMixin):
     file_path = ('data', 'suite')
 
+    def setUp(self):
+        update_toggle_cache(MODULE_FILTER.slug, 'skelly', True, NAMESPACE_DOMAIN)
+        update_toggle_cache(MODULE_FILTER.slug, 'domain', True, NAMESPACE_DOMAIN)
+        update_toggle_cache(MODULE_FILTER.slug, 'example', True, NAMESPACE_DOMAIN)
```
Comment (benrudolph) [inline corehq/apps/app_manager/tests/test_suite.py]: I realize this is very fragile, but I'm hoping we can take it out in two weeks. if there's a better way to do this, let me know

**Comment by snopoke** [comment]
looks good, waiting for tests

File: corehq/apps/app_manager/tests/test_suite.py [hunk 2]
```diff
@@ -766,6 +782,11 @@ class TestFormLinking(SimpleTestCase, TestFileMixin):
             }
         ]
     }
+    def setUp(self):
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_suite.py]: E301 expected 1 blank line, found 0

**Comment by benrudolph** [comment]
Ok this should be working. stopped being lazy and ran the entire app_manager suite instead of just a portion

**Comment by benrudolph** [comment]
:white_check_mark:

**Pull request merged**